### DISCUSSION TITLE
how to use sqlc without context ?

### DISCUSSION THREAD

**Discussion by topben** [sqlc-dev/sqlc#1837] [Q&A] [answered]
hi guys: I have a cron job to do ( fetch third party data and save it to DB)
and have to write to the postgres database.
sqlc requires ctx context.Context but i am not writing an api for a client to call. 
that means no request context via browser. all i want to do is simply write the fetched data to database.
I guess i have three options: 
1. just remove the ctx of the generated codes 
2. fake a useless ctx in order to write things into db 
3. use other golang tools and directly write sql statement (any recommendation ?
is there any better solution ?  or i am just wrong with something ?

**Comment by kyleconroy** [comment, marked as answer]
Contexts are very easy to create. A simple `context.Background()` will suffice.

**Reply by topben** [reply]
got it , thank you!
